addons/microsoft_calendar/tests/test_microsoft_service.py

```diff
@@ -28,7 +28,10 @@ def setUp(self):
         self.fake_next_sync_token_url = f"https://graph.microsoft.com/v1.0/me/calendarView/delta?$deltatoken={self.fake_next_sync_token}"
 
         self.header_prefer = 'outlook.body-content-type="html", odata.maxpagesize=50'
-        self.header = {'Content-type': 'application/json', 'Authorization': 'Bearer %s' % self.fake_token}
+        self.delete_header = {
+            'Authorization': 'Bearer %s' % self.fake_token,
+        }
+        self.header = {'Content-type': 'application/json', **self.delete_header }
```

Comment:
The linter is complaining about the extra space
```suggestion
        self.header = {'Content-type': 'application/json', **self.delete_header}
```